app/views/cases/offender_sar/_closure_outcomes_form.html.slim

```diff
@@ -7,9 +7,6 @@
   .js-info-held-status
     = f.radio_button_fieldset :info_held_status_abbreviation, choices: CaseClosure::InfoHeldStatus.all, text_method: :name, value_method: :abbreviation
 
-  .js-outcome-group
-    = f.radio_button_fieldset :outcome_abbreviation, choices: CaseClosure::Outcome.active, text_method: :name, value_method: :abbreviation
-
```

Comment:
For things like FOI requests, there are partial responses and loads of criteria that need to be completed on closing a case. Offender SAR don't need these so removed from the view